name-service/js/src/bindings.ts

```diff
@@ -217,3 +218,49 @@ export async function deleteNameRegistry(
 
   return changeAuthoritiesInstr;
 }
+
+/**
+ * Realloc the name account space.
+ *
+ * @param connection The solana connection object to the RPC node
+ * @param name The name of the name account
+ * @param space The new space to be allocated
+ * @param payerKey The allocation cost payer if new space is larger than current or the refund destination if smaller
+ * @param nameClass The class of this name, if it exsists
+ * @param nameParent The parent name of this name, if it exists
+ * @returns
+ */
+export async function reallocNameAccount(
+  connection: Connection,
+  name: string,
+  space: number,
+  payerKey: PublicKey,
+  nameClass?: PublicKey,
+  nameParent?: PublicKey
+): Promise<TransactionInstruction> {
+  const hashed_name = await getHashedName(name);
```

Comment:
nit: i know this predates your PR, but in JS, let's stick with camelCase
```suggestion
  const hashedName = await getHashedName(name);
```

Reply:
Good catch! Seeing JS in snake_case hurts my eyes! Switching back and forth from TS and rust really messes with the mind :rofl:

Reply:
That can be done in follow-up work, no need to do it here